.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,6 +33,7 @@ Fix #XXX
 - [ ] Documentation <!-- Adding a new feature? Do you need to document it the README.md or otherwise? -->
 - [ ] [Adheres to Kong style guide](https://github.com/Kong/docs.konghq.com/blob/master/STYLEGUIDE.md)
 - [ ] Spellchecked my updates
+- [ ] Tagged "Team Docs" as reviewers
```

Comment:
Wouldn't this entail tagging us for non-Enterprise reviews as well?

Reply:
I think that is OK - it can help get simple typo type fixes through, or let us ping appropriate core members to help get it through